apps/ledger-live-mobile/src/components/DeviceJob/types.ts

```diff
@@ -1,18 +1,20 @@
 import type { Observable } from "rxjs";
 import type { Device } from "@ledgerhq/live-common/hw/actions/types";
+import { Theme } from "../../colors";
 // meta object are accumulated over steps
 export type Step = {
   Body: React.ComponentType<{
```

Comment:
My understanding is that `React.ComponentType` is specific to Flow but should not be used anymore as we want to get rid of it